pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -106,39 +116,28 @@ public static void updateOrderStatus(BaseMercuryStatusMessage baseMercuryStatusM
                             null, statusDate, null,
                             rs.getLong(DBConstants.TISSUE_ID), rs.getLong(DBConstants.DSM_KIT_REQUEST_ID), null));
                 } else {
-                    dbVals.resultException = new RuntimeException(
-                            "Couldn't get ddp instance id for order " + mercuryStatusMessage.getOrderID());
+                    dbVals.resultException = new DsmInternalError(String.format("Couldn't get ddp instance id for order %s",
```

Comment:
Is this actually a user input error? Where does the order ID come from?
Also, is this error text correct? How does the DDP instance figure in here?

Reply:
No it's not! great catch!! Not sure how that made it's way to here.... I will change it!